### PR TITLE
Dev

### LINKED ISSUE #129: Number of hits with insufficient evidence
Looks like transcript sequences will always have 0 number of hits with insufficient evidence because of the ORF validation that does not require any hits. Here's an [example](http://antgenomes.org/~priyam/GV_2018_07_07_19_09_36). I think `num_hits_with_insufficient_blast_hits` might be more relevant.

### PR DESCRIPTION
**Pull request by IsmailM**
Fixes #129 

adds `--min_blast_hits` (default is 5 as before - shall we change this to 10?)

### CODE REVIEW

File: lib/genevalidator/output.rb [hunk 1]
```diff
@@ -238,7 +238,7 @@ def generate_summary_data(overview)
           ['num_predictions', overview[:no_queries]],
           ['num_good_predictions', overview[:good_scores]],
           ['num_bad_predictions', overview[:bad_scores]],
-          ['num_predictions_with_insufficient_evidence', overview[:nee]],
+          ['num_predictions_with_insufficient_evidence', overview[:insufficient_BLAST_hits]],
```
Comment (yeban) [inline lib/genevalidator/output.rb]: Shouldn't the key be changed to `num_hits_with_insufficient_blast_hits`?

Comment (IsmailM) [reply lib/genevalidator/output.rb]: `num_predictions_with_insufficient_blast_hits`

Comment (IsmailM) [reply lib/genevalidator/output.rb]: >

Comment (yeban) [reply lib/genevalidator/output.rb]: Right

**Pull request merged**